.github/CONTRIBUTING.md

```diff
@@ -69,102 +67,142 @@ There are few ways you can play with the build:
 -  Playing with `webchat-loader`
    -  Navigate to https://compulim.github.io/webchat-loader/
```

Comment:
Add link

Reply:
There should be documentation on how to use the loader

Reply:
Linkified + more docs.